<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Bioinformatic Tools by LABIO@PUCRS</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Bioinformatic Tools</h1>
      <h2 class="project-tagline">Lista de ferramentas utilizadas no ensino de Bioinformática</h2>
      <a href="https://labio-pucrs.github.io/bioinformatic_tools" class="btn">Home</a>
      <a href="https://labio-pucrs.github.io/bioinformatic_tools/classificacao.html" class="btn">Classificação</a>
    </section>

    <section class="main-content">
 <p class=3DMsoNormal style=3D'margin-top:12.0pt;page-break-after:avoid'><b
style=3D'mso-bidi-font-weight:normal'><span lang=3DPT-BR style=3D'font-size=
:13.0pt;
font-family:"Times New Roman",serif;mso-ansi-language:PT-BR'>Referencias <o=
:p></o:p></span></b></p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><!--[if supportFields]><span
style=3D'mso-element:field-begin'></span><span lang=3DPT-BR style=3D'mso-an=
si-language:
PT-BR'><span style=3D'mso-spacerun:yes'>&nbsp;</span>ADDIN ZOTERO_BIBL
{&quot;custom&quot;:[]} CSL_BIBLIOGRAPHY </span><span style=3D'mso-element:=
field-separator'></span><![endif]--><span
class=3DSpellE><span lang=3DPT-BR style=3D'mso-ansi-language:PT-BR'>Alcanta=
ra</span></span><span
lang=3DPT-BR style=3D'mso-ansi-language:PT-BR'>, R., <span class=3DSpellE>O=
nwubiko</span>,
J., <span class=3DSpellE>Cao</span>, H., Matos, P. d., <span class=3DSpellE=
>Cham</span>,
J. A., <span class=3DSpellE>Jacobsen</span>, J., &#8230; </span>Steinbeck, =
C.
(2013). The EBI enzyme portal. <i>Nucleic Acids Research</i>, <i>41</i>(D1),
D773&#8211;D780. http://doi.org/10.1093/nar/gks1112</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Altman, R. B. (1998). A curricul=
um
for bioinformatics: the time is ripe. <i>Bioinformatics (Oxford, England)</=
i>, <i>14</i>(7),
549&#8211;550.</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Atwood, T. K., <span class=3DSpe=
llE>Bongcam-Rudloff</span>,
E., <span class=3DSpellE>Brazas</span>, M. E., <span class=3DSpellE>Corpas<=
/span>,
M., <span class=3DSpellE>Gaudet</span>, P., <span class=3DSpellE>Lewitter</=
span>, F.,
&#8230; Consortium, G. (2015). GOBLET: The Global <span class=3DSpellE>Orga=
nisation</span>
for Bioinformatics Learning, Education and Training. <i>PLOS <span
class=3DSpellE>Comput</span> <span class=3DSpellE>Biol</span></i>, <i>11</i=
>(4),
e1004143. http://doi.org/10.1371/journal.pcbi.1004143</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Bairoch</sp=
an>,
A., <span class=3DSpellE>Apweiler</span>, R., Wu, C. H., Barker, W. C., <sp=
an
class=3DSpellE>Boeckmann</span>, B., Ferro, S., &#8230; <span class=3DSpell=
E>Yeh</span>,
L.-S. L. (2005). The Universal Protein Resource (<span class=3DSpellE>UniPr=
ot</span>).
<i>Nucleic Acids Research</i>, <i>33</i>(Database issue), D154&#8211;D159.
http://doi.org/10.1093/nar/gki070</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Barker, W. C., <span class=3DSpe=
llE>Garavelli</span>,
J. S., Huang, H., <span class=3DSpellE>McGarvey</span>, P. B., <span
class=3DSpellE>Orcutt</span>, B. C., <span class=3DSpellE>Srinivasarao</spa=
n>, G.
Y., &#8230; Wu, C. (2000). The Protein Information Resource (PIR). <i>Nucle=
ic
Acids Research</i>, <i>28</i>(1), 41&#8211;44.
http://doi.org/10.1093/nar/28.1.41</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Benson, D. A., Clark, K., <span
class=3DSpellE>Karsch</span>-Mizrachi, I., <span class=3DSpellE>Lipman</spa=
n>, D.
J., <span class=3DSpellE>Ostell</span>, J., &amp; Sayers, E. W. (2015). Gen=
Bank. <i>Nucleic
Acids Research</i>, <i>43</i>(Database issue), D30&#8211;D35.
http://doi.org/10.1093/nar/gku1216</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Berman, H. M. (2000). The Protein
Data Bank. <i>Nucleic Acids Research</i>, <i>28</i>(1), 235&#8211;242.
http://doi.org/10.1093/nar/28.1.235</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Berman, H. M., Westbrook, J., Fe=
ng,
Z., Gilliland, G., Bhat, T. N., <span class=3DSpellE>Weissig</span>, H., &#=
8230;
Bourne, P. E. (2000). The Protein Data Bank. <i>Nucleic Acids Research</i>,=
 <i>28</i>(1),
235&#8211;242. http://doi.org/10.1093/nar/28.1.235</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Bishop, &Ouml;. T., <span
class=3DSpellE>Adebiyi</span>, E. F., <span class=3DSpellE>Alzohairy</span>=
, A. M.,
Everett, D., <span class=3DSpellE>Ghedira</span>, K., <span class=3DSpellE>=
Ghouila</span>,
A., &#8230; Consortium),<span style=3D'mso-spacerun:yes'>&nbsp; </span>as m=
embers
of T. H. (2014). Bioinformatics Education&#8212;Perspectives and Challenges=
 out
of Africa. <i>Briefings in Bioinformatics</i>, bbu022.
http://doi.org/10.1093/bib/bbu022</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Buchan, D. W. A., <span
class=3DSpellE>Minneci</span>, F., Nugent, T. C. O., Bryson, K., &amp; Jone=
s, D.
T. (2013). Scalable web services for the PSIPRED Protein Analysis Workbench=
. <i>Nucleic
Acids Research</i>, <i>41</i>(Web Server issue), W349&#8211;W357.
http://doi.org/10.1093/nar/gkt381</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Chang, J.-M., Di <span class=3DS=
pellE>Tommaso</span>,
P., &amp; <span class=3DSpellE>Notredame</span>, C. (2014). TCS: a new mult=
iple
sequence alignment reliability measure to estimate alignment accuracy and
improve phylogenetic tree reconstruction. <i>Molecular Biology and Evolutio=
n</i>,
<i>31</i>(6), 1625&#8211;1637. http://doi.org/10.1093/molbev/msu117</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>de Castro, E., <span class=3DSpe=
llE>Sigrist</span>,
C. J. A., <span class=3DSpellE>Gattiker</span>, A., <span class=3DSpellE>Bu=
lliard</span>,
V., <span class=3DSpellE>Langendijk-Genevaux</span>, P. S., <span class=3DS=
pellE>Gasteiger</span>,
E., &#8230; <span class=3DSpellE>Hulo</span>, N. (2006). <span class=3DSpel=
lE>ScanProsite</span>:
detection of PROSITE signature matches and <span class=3DSpellE>ProRule</sp=
an>-associated
functional and structural residues in proteins. <i>Nucleic Acids Research</=
i>, <i>34</i>(Web
Server issue), W362-365. http://doi.org/10.1093/nar/gkl124</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE><span lang=
=3DPT-BR
style=3D'mso-ansi-language:PT-BR'>Dimitrov</span></span><span lang=3DPT-BR
style=3D'mso-ansi-language:PT-BR'>, R. A., &amp; <span class=3DSpellE>Gouli=
amova</span>,
D. E. (2009). </span>Bioinformatics Education: Perspectives and Challenges.=
 <i>Biotechnology
&amp; Biotechnological Equipment</i>, <i>23</i>(sup1), 40&#8211;42.
http://doi.org/10.1080/13102818.2009.10818360</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Drozdetskiy=
</span>,
A., Cole, C., Procter, J., &amp; Barton, G. J. (2015). JPred4: a protein
secondary structure prediction server. <i>Nucleic Acids Research</i>, gkv33=
2.
http://doi.org/10.1093/nar/gkv332</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Edgar, R. C. (2004). MUSCLE: a
multiple sequence alignment method with reduced time and space complexity. =
<i>BMC
Bioinformatics</i>, <i>5</i>, 113. http://doi.org/10.1186/1471-2105-5-113</=
p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Eisenberg, D., <span class=3DSpe=
llE>L&uuml;thy</span>,
R., &amp; Bowie, J. U. (1997). VERIFY3D: assessment of protein models with
three-dimensional profiles. <i>Methods in Enzymology</i>, <i>277</i>,
396&#8211;404.</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Fiser, A., &amp; <span class=3DS=
pellE>&#352;ali</span>,
A. (2003). <span class=3DSpellE>Modeller</span>: Generation and Refinement =
of
Homology-Based Protein Structure Models. In J. and R. M. S. Charles W. Cart=
er
(Org.), <i>Methods in Enzymology</i> (Vol. <span lang=3DPT-BR style=3D'mso-=
ansi-language:
PT-BR'>Volume 374, p. 461&#8211;491). <span class=3DSpellE>Academic</span> =
Press.
Recuperado de http://www.sciencedirect.com/science/article/pii/S00766879037=
40208<o:p></o:p></span></p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span lang=3DPT-BR style=3D'mso-=
ansi-language:
PT-BR'>Gibas, C., &amp; Jambeck, P. (2001). <i>Desenvolvendo
Bioinform&aacute;tica: ferramentas de software para aplica&ccedil;&otilde;e=
s em
biologia</i>. Campus.<o:p></o:p></span></p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span lang=3DPT-BR style=3D'mso-=
ansi-language:
PT-BR'>Gomes, A. S., &amp; Padovani, S. (2005). Usabilidade no ciclo de
desenvolvimento de software educativo. <i>Simp&oacute;sio Brasileiro de Inf=
orm&aacute;tica
na Educa&ccedil;&atilde;o SBIE</i>, <i>2005</i>. Recuperado de
https://www.researchgate.net/profile/Alex_Gomes2/publication/228971486_Usab=
ilidade_no_ciclo_de_desenvolvimento_de_software_educativo/links/00b4951a0f0=
502cb25000000.pdf<o:p></o:p></span></p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Guex</span>=
, N.
(1996). Swiss-<span class=3DSpellE>PdbViewer</span>: A new fast and easy to=
 use
PDB viewer for the Macintosh. <span class=3DSpellE><i>Experientia</i></span=
>, <i>52</i>,
A26.</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Guex</span>=
, N.,
&amp; <span class=3DSpellE>Peitsch</span>, M. C. (1997). SWISS-MODEL and the
Swiss-<span class=3DSpellE>PdbViewer</span>: an environment for comparative
protein modeling. <i>Electrophoresis</i>, <i>18</i>(15), 2714&#8211;2723.
http://doi.org/10.1002/elps.1150181505</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Hogue, C. W. (1997). Cn3D: a new
generation of three-dimensional molecular structure viewer. <i>Trends in
Biochemical Sciences</i>, <i>22</i>(8), 314&#8211;316.</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Hooft</span=
>, R.
W., <span class=3DSpellE>Vriend</span>, G., Sander, C., &amp; <span class=
=3DSpellE>Abola</span>,
E. E. (1996). Errors in protein structures. <i>Nature</i>, <i>381</i>(6580),
272. http://doi.org/10.1038/381272a0</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Humphrey, W., <span class=3DSpel=
lE>Dalke</span>,
A., &amp; <span class=3DSpellE>Schulten</span>, K. (1996). VMD: Visual mole=
cular
dynamics. <i>Journal of Molecular Graphics</i>, <i>14</i>(1), 33&#8211;38.
http://doi.org/10.1016/0263-7855(96)00018-5</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Johnson, M., <span class=3DSpell=
E>Zaretskaya</span>,
I., <span class=3DSpellE>Raytselis</span>, Y., <span class=3DSpellE>Merezhu=
k</span>,
Y., McGinnis, S., &amp; Madden, T. L. (2008). NCBI BLAST: a better web
interface. <i>Nucleic Acids Research</i>, <i>36</i>(Web Server issue), W5-9.
http://doi.org/10.1093/nar/gkn201</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE><span lang=
=3DPT-BR
style=3D'mso-ansi-language:PT-BR'>Kanehisa</span></span><span lang=3DPT-BR
style=3D'mso-ansi-language:PT-BR'>, Minoru, Sato, Yoko, <span class=3DSpell=
E>Kawashima</span>,
<span class=3DSpellE>Masayuki</span>, <span class=3DSpellE>Furumichi</span>=
, <span
class=3DSpellE>Miho</span>, &amp; <span class=3DSpellE>Tanabe</span>, Mao. =
</span>(2016,
<span class=3DSpellE>maio</span> 22). KEGG as a reference resource for gene=
 and
protein annotation. <span lang=3DPT-BR style=3D'mso-ansi-language:PT-BR'>Re=
cuperado
22 de maio de 2016, de
http://nar.oxfordjournals.org/content/44/D1/D457.full?sid=3Dd4ea1e3c-14c6-4=
aa8-82c7-a01737b43261<o:p></o:p></span></p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span lang=3DPT-BR style=3D'mso-=
ansi-language:
PT-BR'>Kumar, S., <span class=3DSpellE>Tamura</span>, K., &amp; Nei, M. (19=
94). </span>MEGA:
Molecular Evolutionary Genetics Analysis software for microcomputers. <i>Co=
mputer
Applications in the Biosciences&#8239;: CABIOS</i>, <i>10</i>(2),
189&#8211;191. http://doi.org/10.1093/bioinformatics/10.2.189</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Larkin, M. A., <span class=3DSpe=
llE>Blackshields</span>,
G., Brown, N. P., <span class=3DSpellE>Chenna</span>, R., <span class=3DSpe=
llE>McGettigan</span>,
P. A., <span class=3DSpellE>McWilliam</span>, H., &#8230; Higgins, D. G. (2=
007). <span
class=3DSpellE>Clustal</span> W and <span class=3DSpellE>Clustal</span> X v=
ersion
2.0. <i>Bioinformatics (Oxford, England)</i>, <i>23</i>(21), 2947&#8211;294=
8.
http://doi.org/10.1093/bioinformatics/btm404</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Laskowski</=
span>,
R. A., MacArthur, M. W., Moss, D. S., &amp; Thornton, J. M. (1993). PROCHEC=
K: a
program to check the <span class=3DSpellE>stereochemical</span> quality of
protein structures. <i>Journal of applied crystallography</i>, <i>26</i>(2),
283&#8211;291.</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Lesk, A. M. (2008). <span
class=3DSpellE><i>Introdu&ccedil;&atilde;o</i></span><i> &agrave;
Bioinform&aacute;tica.</i> <span lang=3DPT-BR style=3D'mso-ansi-language:PT=
-BR'>(2<sup>o</sup>).
Tradu&ccedil;&atilde;o de <span class=3DSpellE>Ardala</span> Elisa Breda An=
drade
et al. LABIO, FACIN, PUCRS. Recuperado de
http://www.ebah.com.br/content/ABAAAfkQ0AG/introducao-a-bioinformatica-arth=
ur-m-lesk-completo<o:p></o:p></span></p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Luscombe, N. M., <span class=3DS=
pellE>Greenbaum</span>,
D., &amp; Gerstein, M. (2001). What is bioinformatics? A proposed definition
and overview of the field. <i>Methods of Information in Medicine</i>, <i>40=
</i>(4),
346&#8211;358.</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE><span lang=
=3DPT-BR
style=3D'mso-ansi-language:PT-BR'>Mart&iacute;-Renom</span></span><span
lang=3DPT-BR style=3D'mso-ansi-language:PT-BR'>, M. A., Stuart, A. C., Fise=
r, A.,
S&aacute;nchez, R., Melo, F., &amp; Sali, A. (2000). </span>Comparative pro=
tein
structure modeling of genes and genomes. <i>Annual Review of Biophysics and
Biomolecular Structure</i>, <i>29</i>, 291&#8211;325. http://doi.org/10.114=
6/annurev.biophys.29.1.291</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Mashima</sp=
an>,
J., Kodama, Y., <span class=3DSpellE>Kosuge</span>, T., Fujisawa, T., Katay=
ama,
T., Nagasaki, H., &#8230; Takagi, T. (2016). DNA data bank of Japan (DDBJ)
progress report. <i>Nucleic Acids Research</i>, <i>44</i>(Database issue),
D51&#8211;D57. http://doi.org/10.1093/nar/gkv1105</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>McWilliam</=
span>,
H., Li, W., <span class=3DSpellE>Uludag</span>, M., <span class=3DSpellE>Sq=
uizzato</span>,
S., Park, Y. M., <span class=3DSpellE>Buso</span>, N., &#8230; Lopez, R. (2=
013).
Analysis Tool Web Services from the EMBL-EBI. <i>Nucleic Acids Research</i>=
, <i>41</i>(Web
Server issue), W597-600. http://doi.org/10.1093/nar/gkt376</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Mount, D. W. (2004). <i>Bioinfor=
matics:
sequence and genome analysis</i>. Cold Spring Harbor Laboratory Press.</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Murzin</spa=
n>, A.
G., Brenner, S. E., Hubbard, T., &amp; <span class=3DSpellE>Chothia</span>,=
 C.
(1995). SCOP: a structural classification of proteins database for the
investigation of sequences and structures. <i>Journal of Molecular Biology<=
/i>,
<i>247</i>(4), 536&#8211;540. http://doi.org/10.1006/jmbi.1995.0159</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Ouzounis</s=
pan>,
C. A., &amp; Valencia, A. (2003). Early bioinformatics: the birth of a
discipline&#8212;a personal view. <i>Bioinformatics</i>, <i>19</i>(17),
2176&#8211;2190. http://doi.org/10.1093/bioinformatics/btg309</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Pettersen</=
span>,
E. F., Goddard, T. D., Huang, C. C., Couch, G. S., Greenblatt, D. M., Meng,=
 E.
C., &amp; <span class=3DSpellE>Ferrin</span>, T. E. (2004). UCSF Chimera&#8=
212;A
visualization system for exploratory research and analysis. <i>Journal of
Computational Chemistry</i>, <i>25</i>(13), 1605&#8211;1612.
http://doi.org/10.1002/jcc.20084</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Pontius, J., <span class=3DSpell=
E>Richelle</span>,
J., &amp; <span class=3DSpellE>Wodak</span>, S. J. (1996). Deviations from
standard atomic volumes as a quality measure for protein crystal structures=
. <i>Journal
of Molecular Biology</i>, <i>264</i>(1), 121&#8211;136. http://doi.org/10.1=
006/jmbi.1996.0628</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Porollo</sp=
an>,
A., &amp; <span class=3DSpellE>Meller</span>, J. (2007). Versatile annotati=
on and
publication quality visualization of protein complexes using POLYVIEW-3D. <=
i>BMC
Bioinformatics</i>, <i>8</i>, 316. http://doi.org/10.1186/1471-2105-8-316</=
p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Ranganathan=
</span>,
S. (2005). Bioinformatics Education&#8212;Perspectives and Challenges. <i>P=
LOS <span
class=3DSpellE>Comput</span> <span class=3DSpellE>Biol</span></i>, <i>1</i>=
(6),
e52. http://doi.org/10.1371/journal.pcbi.0010052</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Reddy, T. B. K., Thomas, A. D.,
Stamatis, D., <span class=3DSpellE>Bertsch</span>, J., <span class=3DSpellE=
>Isbandi</span>,
M., <span class=3DSpellE>Jansson</span>, J., &#8230; <span class=3DSpellE>K=
yrpides</span>,
N. C. (2014). The Genomes <span class=3DSpellE>OnLine</span> Database (GOLD=
) v.5:
a metadata management system based on a four level (meta)genome project
classification. <i>Nucleic Acids Research</i>, gku950.
http://doi.org/10.1093/nar/gku950</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Rezende</sp=
an>,
P. H., &amp; Eloy, M. (2013). <span lang=3DPT-BR style=3D'mso-ansi-language=
:PT-BR'>Sistema
Web para Apoio ao Ensino de Biologia Molecular e Bioinform&aacute;tica. </s=
pan>In
<i>International Conference on Interactive Computer aided Blended Learning
(ICBL2013)</i>. <span lang=3DPT-BR style=3D'mso-ansi-language:PT-BR'>Recupe=
rado de
https://www.icbl-conference.org/proceedings/2013/papers/Contribution63_a.pd=
f<o:p></o:p></span></p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Rose, P. W., <span class=3DSpell=
E>Prli&#263;</span>,
A., Bi, C., <span class=3DSpellE>Bluhm</span>, W. F., Christie, C. H., Dutt=
a, S.,
&#8230; Burley, S. K. (2015). The RCSB Protein Data Bank: views of structur=
al
biology for basic and applied research and education. <i>Nucleic Acids Rese=
arch</i>,
<i>43</i>(Database issue), D345-356. http://doi.org/10.1093/nar/gku1214</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span lang=3DPT-BR style=3D'mso-=
ansi-language:
PT-BR'>Santos Filho, O. A., &amp; Alencastro, R. B. de. (2003). Modelagem de
prote&iacute;nas por homologia. </span><span class=3DSpellE><i>Qu&iacute;mi=
ca</i></span><i>
Nova</i>, <i>26</i>(2), 253&#8211;259.</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Sayle</span=
>, R.
A., &amp; Milner-White, E. J. (1995). RASMOL: biomolecular graphics for all=
. <i>Trends
in Biochemical Sciences</i>, <i>20</i>(9), 374.</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Schr&ouml;dinger, LLC. (2010, <s=
pan
class=3DSpellE>agosto</span>). <i>The <span class=3DSpellE>PyMOL</span> Mol=
ecular
Graphics System, Version 1.3r1</i>.</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>Sillitoe</s=
pan>,
I., Lewis, T. E., Cuff, A., Das, S., Ashford, P., Dawson, N. L., &#8230; <s=
pan
class=3DSpellE>Orengo</span>, C. A. (2015). CATH: comprehensive structural =
and
functional annotations for genome sequences. <i>Nucleic Acids Research</i>,=
 <i>43</i>(D1),
D376&#8211;D381. http://doi.org/10.1093/nar/gku947</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Thompson, J. D., Higgins, D. G.,
&amp; Gibson, T. J. (1994). CLUSTAL W: improving the sensitivity of progres=
sive
multiple sequence alignment through sequence weighting, position-specific g=
ap
penalties and weight matrix choice. <i>Nucleic Acids Research</i>, <i>22</i=
>(22),
4673&#8211;4680.</p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span class=3DSpellE>UniProt</sp=
an>: a
hub for protein information. (2016, <span class=3DSpellE>maio</span> 22). <=
span
lang=3DPT-BR style=3D'mso-ansi-language:PT-BR'>Recuperado 22 de maio de 201=
6, de
http://nar.oxfordjournals.org/content/43/D1/D204.full<o:p></o:p></span></p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'><span lang=3DPT-BR style=3D'mso-=
ansi-language:
PT-BR'>Valente, J. A., &amp; <span class=3DSpellE>others</span>. (1999).
An&aacute;lise dos diferentes tipos de softwares usados na
Educa&ccedil;&atilde;o. <i>O computador na sociedade do conhecimento</i>.
Recuperado de
http://usuarios.upf.br/~teixeira/livros/computador-sociedade-conhecimento.p=
df<o:p></o:p></span></p>

<p class=3DMsoBibliography style=3D'margin-top:0in;margin-right:0in;margin-=
bottom:
6.0pt;margin-left:.5in;line-height:normal'>Welch, L. R., Schwartz, R., &amp=
; <span
class=3DSpellE>Lewitter</span>, F. (2012). A Report of the Curriculum Task =
Force
of the ISCB Education Committee. <i>PLOS <span class=3DSpellE>Comput</span>=
 <span
class=3DSpellE>Biol</span></i>, <i>8</i>(6), e1002570.
http://doi.org/10.1371/journal.pcbi.1002570</p>



      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/labio-pucrs/bioinformatic_tools">Bioinformatic Tools</a> is maintained by <a href="https://github.com/labio-pucrs">labio-pucrs</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
