<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Bioinformatic Tools by LABIO@PUCRS</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Bioinformatic Tools</h1>
      <h2 class="project-tagline">Lista de ferramentas utilizadas no ensino de Bioinformática</h2>
      <a href="https://labio-pucrs.github.io/bioinformatic_tools" class="btn">Home</a>
      <a href="https://labio-pucrs.github.io/bioinformatic_tools/classificacao.html" class="btn">Classificação</a>
    </section>

    <section class="main-content">
 <p>Alcantara, R., Onwubiko, J., Cao, H., Matos, P. d., Cham, J. A., Jacobsen, J., &hellip; Steinbeck, C. (2013). The EBI enzyme portal. <em>Nucleic Acids Research</em>, <em>41</em>(D1), D773&ndash;D780. http://doi.org/10.1093/nar/gks1112</p>
<p>Bairoch, A., Apweiler, R., Wu, C. H., Barker, W. C., Boeckmann, B., Ferro, S., &hellip; Yeh, L.-S. L. (2005). The Universal Protein Resource (UniProt). <em>Nucleic Acids Research</em>, <em>33</em>(Database issue), D154&ndash;D159. http://doi.org/10.1093/nar/gki070</p>
<p>Barker, W. C., Garavelli, J. S., Huang, H., McGarvey, P. B., Orcutt, B. C., Srinivasarao, G. Y., &hellip; Wu, C. (2000). The Protein Information Resource (PIR). <em>Nucleic Acids Research</em>, <em>28</em>(1), 41&ndash;44. http://doi.org/10.1093/nar/28.1.41</p>
<p>Benson, D. A., Clark, K., Karsch-Mizrachi, I., Lipman, D. J., Ostell, J., &amp; Sayers, E. W. (2015). GenBank. <em>Nucleic Acids Research</em>, <em>43</em>(Database issue), D30&ndash;D35. http://doi.org/10.1093/nar/gku1216</p>
<p>Berman, H. M. (2000). The Protein Data Bank. <em>Nucleic Acids Research</em>, <em>28</em>(1), 235&ndash;242. http://doi.org/10.1093/nar/28.1.235</p>
<p>Buchan, D. W. A., Minneci, F., Nugent, T. C. O., Bryson, K., &amp; Jones, D. T. (2013). Scalable web services for the PSIPRED Protein Analysis Workbench. <em>Nucleic Acids Research</em>, <em>41</em>(Web Server issue), W349&ndash;W357. http://doi.org/10.1093/nar/gkt381</p>
<p>Chang, J.-M., Di Tommaso, P., &amp; Notredame, C. (2014). TCS: a new multiple sequence alignment reliability measure to estimate alignment accuracy and improve phylogenetic tree reconstruction. <em>Molecular Biology and Evolution</em>, <em>31</em>(6), 1625&ndash;1637. http://doi.org/10.1093/molbev/msu117</p>
<p>de Castro, E., Sigrist, C. J. A., Gattiker, A., Bulliard, V., Langendijk-Genevaux, P. S., Gasteiger, E., &hellip; Hulo, N. (2006). ScanProsite: detection of PROSITE signature matches and ProRule-associated functional and structural residues in proteins. <em>Nucleic Acids Research</em>, <em>34</em>(Web Server issue), W362&ndash;365. http://doi.org/10.1093/nar/gkl124</p>
<p>Drozdetskiy, A., Cole, C., Procter, J., &amp; Barton, G. J. (2015). JPred4: a protein secondary structure prediction server. <em>Nucleic Acids Research</em>, gkv332. http://doi.org/10.1093/nar/gkv332</p>
<p>Edgar, R. C. (2004). MUSCLE: a multiple sequence alignment method with reduced time and space complexity. <em>BMC Bioinformatics</em>, <em>5</em>, 113. http://doi.org/10.1186/1471-2105-5-113</p>
<p>Eisenberg, D., L&uuml;thy, R., &amp; Bowie, J. U. (1997). VERIFY3D: assessment of protein models with three-dimensional profiles. <em>Methods in Enzymology</em>, <em>277</em>, 396&ndash;404.</p>
<p>Fiser, A., &amp; &Scaron;ali, A. (2003). Modeller: Generation and Refinement of Homology-Based Protein Structure Models. In J. and R. M. S. Charles W. Carter (Org.), <em>Methods in Enzymology</em> (Vol. Volume 374, p. 461&ndash;491). Academic Press. Recuperado de http://www.sciencedirect.com/science/article/pii/S0076687903740208</p>
<p>Guex, N. (1996). Swiss-PdbViewer: A new fast and easy to use PDB viewer for the Macintosh. <em>Experientia</em>, <em>52</em>, A26.</p>
<p>Guex, N., &amp; Peitsch, M. C. (1997). SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modeling. <em>Electrophoresis</em>, <em>18</em>(15), 2714&ndash;2723. http://doi.org/10.1002/elps.1150181505</p>
<p>Hogue, C. W. (1997). Cn3D: a new generation of three-dimensional molecular structure viewer. <em>Trends in Biochemical Sciences</em>, <em>22</em>(8), 314&ndash;316.</p>
<p>Hooft, R. W., Vriend, G., Sander, C., &amp; Abola, E. E. (1996). Errors in protein structures. <em>Nature</em>, <em>381</em>(6580), 272. http://doi.org/10.1038/381272a0</p>
<p>Humphrey, W., Dalke, A., &amp; Schulten, K. (1996). VMD: Visual molecular dynamics. <em>Journal of Molecular Graphics</em>, <em>14</em>(1), 33&ndash;38. http://doi.org/10.1016/0263-7855(96)00018-5</p>
<p>Johnson, M., Zaretskaya, I., Raytselis, Y., Merezhuk, Y., McGinnis, S., &amp; Madden, T. L. (2008). NCBI BLAST: a better web interface. <em>Nucleic Acids Research</em>, <em>36</em>(Web Server issue), W5&ndash;9. http://doi.org/10.1093/nar/gkn201</p>
<p>Kanehisa, Minoru, Sato, Yoko, Kawashima, Masayuki, Furumichi, Miho, &amp; Tanabe, Mao. (2016, maio 22). KEGG as a reference resource for gene and protein annotation. Recuperado 22 de maio de 2016, de http://nar.oxfordjournals.org/content/44/D1/D457.full?sid=d4ea1e3c-14c6-4aa8-82c7-a01737b43261</p>
<p>Kumar, S., Tamura, K., &amp; Nei, M. (1994). MEGA: Molecular Evolutionary Genetics Analysis software for microcomputers. <em>Computer Applications in the Biosciences : CABIOS</em>, <em>10</em>(2), 189&ndash;191. http://doi.org/10.1093/bioinformatics/10.2.189</p>
<p>Larkin, M. A., Blackshields, G., Brown, N. P., Chenna, R., McGettigan, P. A., McWilliam, H., &hellip; Higgins, D. G. (2007). Clustal W and Clustal X version 2.0. <em>Bioinformatics (Oxford, England)</em>, <em>23</em>(21), 2947&ndash;2948. http://doi.org/10.1093/bioinformatics/btm404</p>
<p>Laskowski, R. A., MacArthur, M. W., Moss, D. S., &amp; Thornton, J. M. (1993). PROCHECK: a program to check the stereochemical quality of protein structures. <em>Journal of applied crystallography</em>, <em>26</em>(2), 283&ndash;291.</p>
<p>Mashima, J., Kodama, Y., Kosuge, T., Fujisawa, T., Katayama, T., Nagasaki, H., &hellip; Takagi, T. (2016). DNA data bank of Japan (DDBJ) progress report. <em>Nucleic Acids Research</em>, <em>44</em>(Database issue), D51&ndash;D57. http://doi.org/10.1093/nar/gkv1105</p>
<p>McWilliam, H., Li, W., Uludag, M., Squizzato, S., Park, Y. M., Buso, N., &hellip; Lopez, R. (2013). Analysis Tool Web Services from the EMBL-EBI. <em>Nucleic Acids Research</em>, <em>41</em>(Web Server issue), W597&ndash;600. http://doi.org/10.1093/nar/gkt376</p>
<p>Murzin, A. G., Brenner, S. E., Hubbard, T., &amp; Chothia, C. (1995). SCOP: a structural classification of proteins database for the investigation of sequences and structures. <em>Journal of Molecular Biology</em>, <em>247</em>(4), 536&ndash;540. http://doi.org/10.1006/jmbi.1995.0159</p>
<p>Pettersen, E. F., Goddard, T. D., Huang, C. C., Couch, G. S., Greenblatt, D. M., Meng, E. C., &amp; Ferrin, T. E. (2004). UCSF Chimera&mdash;A visualization system for exploratory research and analysis. <em>Journal of Computational Chemistry</em>, <em>25</em>(13), 1605&ndash;1612. http://doi.org/10.1002/jcc.20084</p>
<p>Pontius, J., Richelle, J., &amp; Wodak, S. J. (1996). Deviations from standard atomic volumes as a quality measure for protein crystal structures. <em>Journal of Molecular Biology</em>, <em>264</em>(1), 121&ndash;136. http://doi.org/10.1006/jmbi.1996.0628</p>
<p>Porollo, A., &amp; Meller, J. (2007). Versatile annotation and publication quality visualization of protein complexes using POLYVIEW-3D. <em>BMC Bioinformatics</em>, <em>8</em>, 316. http://doi.org/10.1186/1471-2105-8-316</p>
<p>Reddy, T. B. K., Thomas, A. D., Stamatis, D., Bertsch, J., Isbandi, M., Jansson, J., &hellip; Kyrpides, N. C. (2014). The Genomes OnLine Database (GOLD) v.5: a metadata management system based on a four level (meta)genome project classification. <em>Nucleic Acids Research</em>, gku950. http://doi.org/10.1093/nar/gku950</p>
<p>Rose, P. W., Prlić, A., Bi, C., Bluhm, W. F., Christie, C. H., Dutta, S., &hellip; Burley, S. K. (2015). The RCSB Protein Data Bank: views of structural biology for basic and applied research and education. <em>Nucleic Acids Research</em>, <em>43</em>(Database issue), D345&ndash;356. http://doi.org/10.1093/nar/gku1214</p>
<p>Sayle, R. A., &amp; Milner-White, E. J. (1995). RASMOL: biomolecular graphics for all. <em>Trends in Biochemical Sciences</em>, <em>20</em>(9), 374.</p>
<p>Schr&ouml;dinger, LLC. (2010, agosto). <em>The PyMOL Molecular Graphics System, Version 1.3r1</em>.</p>
<p>Sillitoe, I., Lewis, T. E., Cuff, A., Das, S., Ashford, P., Dawson, N. L., &hellip; Orengo, C. A. (2015). CATH: comprehensive structural and functional annotations for genome sequences. <em>Nucleic Acids Research</em>, <em>43</em>(D1), D376&ndash;D381. http://doi.org/10.1093/nar/gku947</p>
<p>UniProt: a hub for protein information. (2016, maio 22). Recuperado 22 de maio de 2016, de http://nar.oxfordjournals.org/content/43/D1/D204.full</p>     <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/labio-pucrs/bioinformatic_tools">Bioinformatic Tools</a> is maintained by <a href="https://github.com/labio-pucrs">labio-pucrs</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
