<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Bioinformatic Tools by LABIO@PUCRS</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Bioinformatic Tools</h1>
      <h2 class="project-tagline">Lista de ferramentas utilizadas no ensino de Bioinformática</h2>
      <a href="https://labio-pucrs.github.io/bioinformatic_tools" class="btn">Home</a>
      <a href="https://labio-pucrs.github.io/bioinformatic_tools/classificacao.html" class="btn">Classificação</a>
    </section>

    <section class="main-content">
 Referencias 
Alcantara, R., Onwubiko, J., Cao, H., Matos, P. d., Cham, J. A., Jacobsen, J., … Steinbeck, C. (2013). The EBI enzyme portal. Nucleic Acids Research, 41(D1), D773–D780. http://doi.org/10.1093/nar/gks1112
Altman, R. B. (1998). A curriculum for bioinformatics: the time is ripe. Bioinformatics (Oxford, England), 14(7), 549–550.
Atwood, T. K., Bongcam-Rudloff, E., Brazas, M. E., Corpas, M., Gaudet, P., Lewitter, F., … Consortium, G. (2015). GOBLET: The Global Organisation for Bioinformatics Learning, Education and Training. PLOS Comput Biol, 11(4), e1004143. http://doi.org/10.1371/journal.pcbi.1004143
Bairoch, A., Apweiler, R., Wu, C. H., Barker, W. C., Boeckmann, B., Ferro, S., … Yeh, L.-S. L. (2005). The Universal Protein Resource (UniProt). Nucleic Acids Research, 33(Database issue), D154–D159. http://doi.org/10.1093/nar/gki070
Barker, W. C., Garavelli, J. S., Huang, H., McGarvey, P. B., Orcutt, B. C., Srinivasarao, G. Y., … Wu, C. (2000). The Protein Information Resource (PIR). Nucleic Acids Research, 28(1), 41–44. http://doi.org/10.1093/nar/28.1.41
Benson, D. A., Clark, K., Karsch-Mizrachi, I., Lipman, D. J., Ostell, J., & Sayers, E. W. (2015). GenBank. Nucleic Acids Research, 43(Database issue), D30–D35. http://doi.org/10.1093/nar/gku1216
Berman, H. M. (2000). The Protein Data Bank. Nucleic Acids Research, 28(1), 235–242. http://doi.org/10.1093/nar/28.1.235
Berman, H. M., Westbrook, J., Feng, Z., Gilliland, G., Bhat, T. N., Weissig, H., … Bourne, P. E. (2000). The Protein Data Bank. Nucleic Acids Research, 28(1), 235–242. http://doi.org/10.1093/nar/28.1.235
Bishop, Ö. T., Adebiyi, E. F., Alzohairy, A. M., Everett, D., Ghedira, K., Ghouila, A., … Consortium),  as members of T. H. (2014). Bioinformatics Education—Perspectives and Challenges out of Africa. Briefings in Bioinformatics, bbu022. http://doi.org/10.1093/bib/bbu022
Buchan, D. W. A., Minneci, F., Nugent, T. C. O., Bryson, K., & Jones, D. T. (2013). Scalable web services for the PSIPRED Protein Analysis Workbench. Nucleic Acids Research, 41(Web Server issue), W349–W357. http://doi.org/10.1093/nar/gkt381
Chang, J.-M., Di Tommaso, P., & Notredame, C. (2014). TCS: a new multiple sequence alignment reliability measure to estimate alignment accuracy and improve phylogenetic tree reconstruction. Molecular Biology and Evolution, 31(6), 1625–1637. http://doi.org/10.1093/molbev/msu117
de Castro, E., Sigrist, C. J. A., Gattiker, A., Bulliard, V., Langendijk-Genevaux, P. S., Gasteiger, E., … Hulo, N. (2006). ScanProsite: detection of PROSITE signature matches and ProRule-associated functional and structural residues in proteins. Nucleic Acids Research, 34(Web Server issue), W362-365. http://doi.org/10.1093/nar/gkl124
Dimitrov, R. A., & Gouliamova, D. E. (2009). Bioinformatics Education: Perspectives and Challenges. Biotechnology & Biotechnological Equipment, 23(sup1), 40–42. http://doi.org/10.1080/13102818.2009.10818360
Drozdetskiy, A., Cole, C., Procter, J., & Barton, G. J. (2015). JPred4: a protein secondary structure prediction server. Nucleic Acids Research, gkv332. http://doi.org/10.1093/nar/gkv332
Edgar, R. C. (2004). MUSCLE: a multiple sequence alignment method with reduced time and space complexity. BMC Bioinformatics, 5, 113. http://doi.org/10.1186/1471-2105-5-113
Eisenberg, D., Lüthy, R., & Bowie, J. U. (1997). VERIFY3D: assessment of protein models with three-dimensional profiles. Methods in Enzymology, 277, 396–404.
Fiser, A., & Šali, A. (2003). Modeller: Generation and Refinement of Homology-Based Protein Structure Models. In J. and R. M. S. Charles W. Carter (Org.), Methods in Enzymology (Vol. Volume 374, p. 461–491). Academic Press. Recuperado de http://www.sciencedirect.com/science/article/pii/S0076687903740208
Gibas, C., & Jambeck, P. (2001). Desenvolvendo Bioinformática: ferramentas de software para aplicações em biologia. Campus.
Gomes, A. S., & Padovani, S. (2005). Usabilidade no ciclo de desenvolvimento de software educativo. Simpósio Brasileiro de Informática na Educação SBIE, 2005. Recuperado de https://www.researchgate.net/profile/Alex_Gomes2/publication/228971486_Usabilidade_no_ciclo_de_desenvolvimento_de_software_educativo/links/00b4951a0f0502cb25000000.pdf
Guex, N. (1996). Swiss-PdbViewer: A new fast and easy to use PDB viewer for the Macintosh. Experientia, 52, A26.
Guex, N., & Peitsch, M. C. (1997). SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modeling. Electrophoresis, 18(15), 2714–2723. http://doi.org/10.1002/elps.1150181505
Hogue, C. W. (1997). Cn3D: a new generation of three-dimensional molecular structure viewer. Trends in Biochemical Sciences, 22(8), 314–316.
Hooft, R. W., Vriend, G., Sander, C., & Abola, E. E. (1996). Errors in protein structures. Nature, 381(6580), 272. http://doi.org/10.1038/381272a0
Humphrey, W., Dalke, A., & Schulten, K. (1996). VMD: Visual molecular dynamics. Journal of Molecular Graphics, 14(1), 33–38. http://doi.org/10.1016/0263-7855(96)00018-5
Johnson, M., Zaretskaya, I., Raytselis, Y., Merezhuk, Y., McGinnis, S., & Madden, T. L. (2008). NCBI BLAST: a better web interface. Nucleic Acids Research, 36(Web Server issue), W5-9. http://doi.org/10.1093/nar/gkn201
Kanehisa, Minoru, Sato, Yoko, Kawashima, Masayuki, Furumichi, Miho, & Tanabe, Mao. (2016, maio 22). KEGG as a reference resource for gene and protein annotation. Recuperado 22 de maio de 2016, de http://nar.oxfordjournals.org/content/44/D1/D457.full?sid=d4ea1e3c-14c6-4aa8-82c7-a01737b43261
Kumar, S., Tamura, K., & Nei, M. (1994). MEGA: Molecular Evolutionary Genetics Analysis software for microcomputers. Computer Applications in the Biosciences : CABIOS, 10(2), 189–191. http://doi.org/10.1093/bioinformatics/10.2.189
Larkin, M. A., Blackshields, G., Brown, N. P., Chenna, R., McGettigan, P. A., McWilliam, H., … Higgins, D. G. (2007). Clustal W and Clustal X version 2.0. Bioinformatics (Oxford, England), 23(21), 2947–2948. http://doi.org/10.1093/bioinformatics/btm404
Laskowski, R. A., MacArthur, M. W., Moss, D. S., & Thornton, J. M. (1993). PROCHECK: a program to check the stereochemical quality of protein structures. Journal of applied crystallography, 26(2), 283–291.
Lesk, A. M. (2008). Introdução à Bioinformática. (2o). Tradução de Ardala Elisa Breda Andrade et al. LABIO, FACIN, PUCRS. Recuperado de http://www.ebah.com.br/content/ABAAAfkQ0AG/introducao-a-bioinformatica-arthur-m-lesk-completo
Luscombe, N. M., Greenbaum, D., & Gerstein, M. (2001). What is bioinformatics? A proposed definition and overview of the field. Methods of Information in Medicine, 40(4), 346–358.
Martí-Renom, M. A., Stuart, A. C., Fiser, A., Sánchez, R., Melo, F., & Sali, A. (2000). Comparative protein structure modeling of genes and genomes. Annual Review of Biophysics and Biomolecular Structure, 29, 291–325. http://doi.org/10.1146/annurev.biophys.29.1.291
Mashima, J., Kodama, Y., Kosuge, T., Fujisawa, T., Katayama, T., Nagasaki, H., … Takagi, T. (2016). DNA data bank of Japan (DDBJ) progress report. Nucleic Acids Research, 44(Database issue), D51–D57. http://doi.org/10.1093/nar/gkv1105
McWilliam, H., Li, W., Uludag, M., Squizzato, S., Park, Y. M., Buso, N., … Lopez, R. (2013). Analysis Tool Web Services from the EMBL-EBI. Nucleic Acids Research, 41(Web Server issue), W597-600. http://doi.org/10.1093/nar/gkt376
Mount, D. W. (2004). Bioinformatics: sequence and genome analysis. Cold Spring Harbor Laboratory Press.
Murzin, A. G., Brenner, S. E., Hubbard, T., & Chothia, C. (1995). SCOP: a structural classification of proteins database for the investigation of sequences and structures. Journal of Molecular Biology, 247(4), 536–540. http://doi.org/10.1006/jmbi.1995.0159
Ouzounis, C. A., & Valencia, A. (2003). Early bioinformatics: the birth of a discipline—a personal view. Bioinformatics, 19(17), 2176–2190. http://doi.org/10.1093/bioinformatics/btg309
Pettersen, E. F., Goddard, T. D., Huang, C. C., Couch, G. S., Greenblatt, D. M., Meng, E. C., & Ferrin, T. E. (2004). UCSF Chimera—A visualization system for exploratory research and analysis. Journal of Computational Chemistry, 25(13), 1605–1612. http://doi.org/10.1002/jcc.20084
Pontius, J., Richelle, J., & Wodak, S. J. (1996). Deviations from standard atomic volumes as a quality measure for protein crystal structures. Journal of Molecular Biology, 264(1), 121–136. http://doi.org/10.1006/jmbi.1996.0628
Porollo, A., & Meller, J. (2007). Versatile annotation and publication quality visualization of protein complexes using POLYVIEW-3D. BMC Bioinformatics, 8, 316. http://doi.org/10.1186/1471-2105-8-316
Ranganathan, S. (2005). Bioinformatics Education—Perspectives and Challenges. PLOS Comput Biol, 1(6), e52. http://doi.org/10.1371/journal.pcbi.0010052
Reddy, T. B. K., Thomas, A. D., Stamatis, D., Bertsch, J., Isbandi, M., Jansson, J., … Kyrpides, N. C. (2014). The Genomes OnLine Database (GOLD) v.5: a metadata management system based on a four level (meta)genome project classification. Nucleic Acids Research, gku950. http://doi.org/10.1093/nar/gku950
Rezende, P. H., & Eloy, M. (2013). Sistema Web para Apoio ao Ensino de Biologia Molecular e Bioinformática. In International Conference on Interactive Computer aided Blended Learning (ICBL2013). Recuperado de https://www.icbl-conference.org/proceedings/2013/papers/Contribution63_a.pdf
Rose, P. W., Prlić, A., Bi, C., Bluhm, W. F., Christie, C. H., Dutta, S., … Burley, S. K. (2015). The RCSB Protein Data Bank: views of structural biology for basic and applied research and education. Nucleic Acids Research, 43(Database issue), D345-356. http://doi.org/10.1093/nar/gku1214
Santos Filho, O. A., & Alencastro, R. B. de. (2003). Modelagem de proteínas por homologia. Química Nova, 26(2), 253–259.
Sayle, R. A., & Milner-White, E. J. (1995). RASMOL: biomolecular graphics for all. Trends in Biochemical Sciences, 20(9), 374.
Schrödinger, LLC. (2010, agosto). The PyMOL Molecular Graphics System, Version 1.3r1.
Sillitoe, I., Lewis, T. E., Cuff, A., Das, S., Ashford, P., Dawson, N. L., … Orengo, C. A. (2015). CATH: comprehensive structural and functional annotations for genome sequences. Nucleic Acids Research, 43(D1), D376–D381. http://doi.org/10.1093/nar/gku947
Thompson, J. D., Higgins, D. G., & Gibson, T. J. (1994). CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Research, 22(22), 4673–4680.
UniProt: a hub for protein information. (2016, maio 22). Recuperado 22 de maio de 2016, de http://nar.oxfordjournals.org/content/43/D1/D204.full
Valente, J. A., & others. (1999). Análise dos diferentes tipos de softwares usados na Educação. O computador na sociedade do conhecimento. Recuperado de http://usuarios.upf.br/~teixeira/livros/computador-sociedade-conhecimento.pdf
Welch, L. R., Schwartz, R., & Lewitter, F. (2012). A Report of the Curriculum Task Force of the ISCB Education Committee. PLOS Comput Biol, 8(6), e1002570. http://doi.org/10.1371/journal.pcbi.1002570


      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/labio-pucrs/bioinformatic_tools">Bioinformatic Tools</a> is maintained by <a href="https://github.com/labio-pucrs">labio-pucrs</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
